content/en/news/releases/1.8.x/announcing-1.8/change-notes/index.md

```diff
@@ -0,0 +1,172 @@
+
+- **Added** Trust Domain Validation by default rejecting requests in sidecars if the request is not from same trust domain
+or if it's not in the `TrustDomainAliases` specified in the `MeshConfig`.
+  ([Issue #26224](https://github.com/istio/istio/issues/26224))
+
+## Telemetry
+
+- **Updated** the "Control Plane Dashboard" and the "Performance Dashboard" to use the `container_memory_working_set_bytes` metric
+to display memory. This metric only counts memory that *cannot be reclaimed* by the kernel even under memory pressure,
+and therefore more relevant for tracking. It is also consistent with `kubectl top`. The reported values are lower than
+the previous values.
+
+- **Added** Update Istio Workload and Istio Service dashboards to improve loading time.
+  ([Issue #22408](https://github.com/istio/istio/issues/22408))
+
+- **Added** `datasource` parameter to Grafana dashboards
+  ([Issue #22408](https://github.com/istio/istio/issues/22408))
+
+- **Added** Listener Access Logs when `ResponseFlag` from Envoy is set.
+  ([Issue #26851](https://github.com/istio/istio/issues/26851))
+
+- **Added** support for `OpenCensusAgent` formatted trace export with configurable trace context headers.
+
+- **Added** Proxy config to control Envoy native stats generation.
+  ([Issue #26546](https://github.com/istio/istio/issues/26546))
+
+- **Added** Istio Wasm Extension Grafana Dashboard.
+  ([Issue #25843](https://github.com/istio/istio/issues/25843))
+
+- **Fixed** client side metric  missing destination workload labels for out-of-mesh destination or request that fails to reach destination.
+  ([Issue #20538](https://github.com/istio/istio/issues/20538))
+
+- **Fixed** interpretation of `$(HOST_IP)` in Zipkin and Datadog tracer address.
+  ([Issue #27911](https://github.com/istio/istio/issues/27911))
+
+- **Removed** all Mixer-related features and functionality. This is a scheduled
+removal of a deprecated Istio services and deployments, as well as
+Mixer-focused CRDs and component and related functionality.
+  ([Issue #25333](https://github.com/istio/istio/issues/25333)),([Issue #24300](https://github.com/istio/istio/issues/24300))
+
+## Installation
+
+- **Improved** multi-network configuration so that labeling a service with `topology.istio.io/network=network-name` can
+configure cross-network gateways without using [mesh networks](/docs/reference/config/istio.mesh.v1alpha1/#MeshNetworks).
+
+- **Improved** sidecar injection to not modify the pod `securityPolicy.fsGroup` which could conflict with existing settings and secret mounts.
+ This option is enabled automatically on Kubernetes 1.19+ and is not supported on older versions.
+  ([Issue #26882](https://github.com/istio/istio/issues/26882))
+
+- **Improved** Generated operator manifests for use with `kustomize` are available in the [manifests]({{</* github_tree */>}}/manifests/charts/istio-operator/files) directory.
```

Comment:
`{{</* github_tree */>}}` looks like it should be `{{< github_tree >}}/samples/sleep)`

https://github.com/istio/istio.io/search?q=github_tree

Reply:
ach, I had copied from the source - fixed